lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -935,10 +935,20 @@ ParseResult FIRParser::parseType(FIRRTLType &result, const Twine &message) {
     auto loc = getToken().getLoc();
     consumeToken();
     FIRRTLType type;
+    SmallVector<StringRef> layers;
 
     if (parseToken(FIRToken::less, "expected '<' in reference type") ||
-        parseType(type, "expected probe data type") ||
-        parseToken(FIRToken::greater, "expected '>' in reference type"))
+        parseType(type, "expected probe data type"))
+      return failure();
+    while (getToken().getKind() == FIRToken::identifier) {
+      StringRef layer;
+      if (parseId(layer, "expected layer name"))
+        return failure();
```

Comment:
This should be dead code given that the `while` is conditioned on executing only if this is an ID.

Reply:
Fixed in dea1a8e379dca787c4b9fe19a1baf9b0ed45c6be.